cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+import dateutil
+
+
+def advance_appliance_date_by_day(appliance):
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
+    appliance_date = dateutil.parser.parse(txt_date)
+    td = datetime.timedelta(days=1)
+    advanced_txt_date = (appliance_date + td).strftime("%Y-%m-%d %H:%M:%S%z")
+    appliance.ssh_client.run_command("date -s '{}'".format(advanced_txt_date))
```

Comment:
I am debating this myself, but would it be good to reset the time back to what it was? @digitronik thoughts?

Reply:
The time is set on temp appliance, so I don't think it is necessary to reset the time back.

Reply:
The time is set on temp appliance, so I don't think it is necessary to reset the time back.

Reply:
@jaryn you are correct here. :+1: